lazy_dataset/core.py

```diff
@@ -2763,6 +2758,88 @@ def __getitem__(self, item):
             return super().__getitem__(item)
 
 
+class TileDataset(Dataset):
+    """
+    Iterates over all elements of the input_dataset for `repetitions` times.
+
+    """
+
+    def __init__(self, input_dataset, repetitions):
```

Comment:
Could you rename `repetitions` to `reps`?
I prefer to have names close to numpy. (We don't want to do the same as pytorch, where you have to learn the new names for arguments, because they differ to numpy)